src/pages/_app.tsx

```diff
@@ -1,6 +1,28 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
+import theme from "@/theme";
+import { ChakraProvider, chakra } from "@chakra-ui/react";
+import type { AppProps } from "next/app";
+import { Barlow, Fraunces } from "next/font/google";
+
+const barlow = Barlow({
+  weight: ["400", "600"],
+  subsets: ["latin"],
+  variable: "--font-barlow",
+});
+
+const fraunces = Fraunces({
+  weight: ["700", "900"],
+  subsets: ["latin"],
+  variable: "--font-fraunces",
+});
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
+    <ChakraProvider theme={theme}>
+      <div className={fraunces.variable}>
```

Comment:
going to leave this as a comment, but see if you can concatenate `barlow & fraunces` into one `div` element.